src/style.css

```diff
@@ -1 +1,60 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url('./fonts/Avenir-Book.ttf') format('truetype'),
+    url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+.form {
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  display: flex;
+}
+
+.big-input {
+  flex: 1;
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  background: #fff url('./images/Search.svg') no-repeat left 26px center;
+  background-size: 19px 19px;
+}
+
+.small-input {
+  flex: 1;
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  background: #fff url('./images/Search.svg') no-repeat left 13px center;
+  background-size: 11px 11px;
+}
+
+input:hover {
```

Comment:
do not add styles using tag selector

Instead, add a common `input` class to both inputs and add all shared styles using this class (instead of repeating them in small-input and biginput)